server.js

```diff
@@ -0,0 +1,24 @@
+import { createFastifyApp } from "./app.js";
+
+const app = createFastifyApp();
+const port = Number(process.env.PORT ?? 3000);
+
+process.on("SIGINT", async () => {
+	app.log.info("Shutting down server...");
+	await app.close();
+	process.exit(0);
+});
+
+const startServer = async () => {
+	try {
+		await app.listen({ port });
+	} catch (err) {
+		app.log.fatal(
+			{ err, port },
```

Comment:
try to see how it prints it
I think switching the order of the params will make it easier to see the port param

```suggestion
			{ port, err },
```